Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,9 +154,12 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
-        while line = read.gets
-          puts line if verbose
-          @output += line
+        while buf = read.read(8)
```

Comment:
This is 8 bytes (rather than bits) I believe so it's still gonna only print every 8th `.` unfortunately.